src/components/Card.js

```diff
@@ -0,0 +1,47 @@
+import noneCardImg from '../assets/Linkbrary.svg';
+import kebab from '../assets/kebab.svg';
+import emptyStar from '../assets/icon-empty-star.svg';
+import fullStar from '../assets/icon-full-star.svg';
+import ElapsedTime from '../utils/ElapsedTime';
+import FormatDate from '../utils/FormatDate';
+import './Card.css';
+
+const Card = ({ link }) => {
+  const { created_at, description, image_source, url } = link;
+  const fillStar = false;
```

Comment:
이런 식으로 추후에 들어올 데이터를 대비해 테스트용으로 작성한 경우, 추후 처리할 수 있도록 주석을 달아두는 게 좋습니다.
```suggestion
  // TODO: 실제 데이터 들어오면 수정 반영할 것
  const fillStar = false;
```